modules/test/playwright/tests/portal-search-web/customFilter.spec.ts

```diff
@@ -82,7 +82,10 @@ test.describe('Custom filter configuration works as expected on content pages',
 		});
 
 		await test.step('Configure custom filter in default experience to return only results that contains Portuguese in the title_en_US field', async () => {
-			await searchPage.openSearchPortletConfiguration('Custom Filter');
+			const widgetId =
+				await pageEditorPage.getFragmentId('Custom Filter');
+
+			await pageEditorPage.clickFragmentOption(widgetId, 'Configuration');
```

Comment:
Can we rename `widgetId` to be more descriptive like `customFilterFragmentId`? 

Not a big deal, but the little things can add up with readability speed. So if I was directly reading `await pageEditorPage.clickFragmentOption(widgetId, 'Configuration');` I'd ask myself "which widget configuration is it clicking?" where I would have to look what `widgetId` is defined as.

Reply:
Good idea! Will add that in 👍